tensorboard/tools/diagnose_tensorboard.py

```diff
@@ -244,6 +256,29 @@ def tensorboard_binary_path():
   logging.info("which tensorboard: %r", which("tensorboard"))
 
 
+@check
+def addrinfos():
+  """Gets family and nodename for a loopback address."""
+  print("On `socket`: %r" % (dir(socket),))
```

Comment:
Printing 4KB of everything on `socket` seems maybe like too much to me - it certainly could be useful for debugging and shouldn't contain anything sensitive, but I could see it causing more hesitation on the part of users since it's such a large blob.  Also, it's not collapsed in the email version of the issue so it does increase the amount we have to scroll through every time.

IMO a bit better just to stick to reading the targeted attributes and expanding that list over time as need be (and if we really need the full dump of `socket` from a user, we can ask for it).

Reply:
Fine with me. I’ll just remove it for now—the `sgetattrs` are printing
what we care about—and, like you say, we can add more later. (I didn’t
realize that the email notifications didn’t render the `<details>`, but
I guess that makes sense.)